superx/models.py

```diff
@@ -0,0 +1,43 @@
+"""Data models."""
+from . import db
+
+
+class User(db.Model):
```

Comment:
class User should be for users, but you are using it for item_info. Maybe the class should be called ItemInfo